Patch from upstream git repo that happens to also solve
Bug 24711806 - meld bus error on SPARC in S12 build #106

From a11de0da6b4fcc59c752c981921f268dee78e7db Mon Sep 17 00:00:00 2001
From: Benjamin Otte <otte@redhat.com>
Date: Tue, 15 Dec 2015 16:03:25 +0100
Subject: [PATCH] stylecontext: Builtin images can be (and are) transformed

So don't special-case them out.
---
 gtk/gtkstylecontext.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/gtk/gtkstylecontext.c b/gtk/gtkstylecontext.c
index 7e8c2a0..65ebe68 100644
--- a/gtk/gtkstylecontext.c
+++ b/gtk/gtkstylecontext.c
@@ -28,7 +28,6 @@
 #include "gtkcsscolorvalueprivate.h"
 #include "gtkcsscornervalueprivate.h"
 #include "gtkcssenumvalueprivate.h"
-#include "gtkcssimagebuiltinprivate.h"
 #include "gtkcssimagevalueprivate.h"
 #include "gtkcssnodedeclarationprivate.h"
 #include "gtkcssnodeprivate.h"
@@ -3091,10 +3090,6 @@ _gtk_style_context_get_icon_extents (GtkStyleContext *context,
   extents->width = width;
   extents->height = height;
 
-  /* builtin images can't be transformed */
-  if (GTK_IS_CSS_IMAGE_BUILTIN (_gtk_style_context_peek_property (context, GTK_CSS_PROPERTY_ICON_SOURCE)))
-    return;
-
   if (!_gtk_css_transform_value_get_matrix (_gtk_style_context_peek_property (context, GTK_CSS_PROPERTY_ICON_TRANSFORM), &transform_matrix))
     return;
   
-- 
2.7.4

